pkg/mcproducer/scripts/pull-image.sh

```diff
@@ -4,10 +4,12 @@
 kernel_module_image_filepath="$KERNEL_MODULE_IMAGE_FILEPATH"
 worker_image="$WORKER_IMAGE"
 kernel_module_image="$KERNEL_MODULE_IMAGE"
+kernel_module_image_tag=$(uname -r)
 
 if [ -e $kernel_module_image_filepath ]; then
-    echo "File $kernel_module_image_filepath found.Nothing to do, the file was handled, removing it"
+    echo "File $kernel_module_image_filepath found.Nothing to do, the file was handled, removing $kernel_module_image_filepath and $kmm_config_file_filepath"
```

Comment:
```suggestion
    echo "File $kernel_module_image_filepath found. Nothing to do, the file was handled, removing $kernel_module_image_filepath and $kmm_config_file_filepath"
```

Reply:
why do we need space after dot?

Reply:
Because it clearer? Took me few seconds to realize it is not a `Nothing` field name in a `found` object :)